xml/System.Diagnostics.Metrics/Meter.xml

```diff
@@ -466,13 +466,21 @@ Example uses for ObservableCounter: The number of page faults for each process.
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="name">To be added.</param>
-        <param name="observeValues">To be added.</param>
-        <param name="unit">To be added.</param>
-        <param name="description">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The instrument name. Cannot be <see langword="null" />.</param>
+        <param name="observeValues">The callback to call to get the measurements when the <see cref="M:System.Diagnostics.Metrics.ObservableUpDownCounter`1.Observe" /> is called by <see cref="M:System.Diagnostics.Metrics.MeterListener.RecordObservableInstruments" />.</param>
```

Comment:
You can't link to ``ObservableUpDownCounter`1.Observe`` either - it's a protected method in a sealed class so it's not on docs.

Reply:
ok, ``ObservableInstrument`1.Observe`` is the only choice then. That is not perfect but acceptable.

Reply:
```suggestion
        <param name="observeValues">The callback to call to get the measurements when the <see cref="M:System.Diagnostics.Metrics.ObservableInstrument`1.Observe" /> is called by <see cref="M:System.Diagnostics.Metrics.MeterListener.RecordObservableInstruments" />.</param>
```